src/ctr_stdio.c

```diff
@@ -129,6 +129,9 @@ static bool read_stdio(int fd, stdpipe_t pipe, gboolean *eof)
 			*eof = true;
 		return false;
 	} else if (num_read < 0) {
+		if (errno == EIO && isatty(fd)){
```

Comment:
really not important, but if we're returning the same value, we could invert this condition and put the warning in here instead. idk if that would make it hard to read though. I think either way, a comment would be helpful

Reply:
I tried to invert it but it looks more difficult to read.

I've added a comment